### DISCUSSION TITLE
I wanted to use the tool but I have an error

### DISCUSSION THREAD

**Discussion by dzGeekyGirl** [danieldevine/bird-elephant#28] [General]
**Describe the bug**
Parse error: syntax error, unexpected '/', expecting ';' or ',

**To Reproduce**
I install the tool using the 
`composer require coderjerk/bird-elephant`
I wanted to try the exemple 
`use Coderjerk/BirdElephant/BirdElephant;

//your credentials, should be passed in via $_ENV or similar, don't hardcode.
$credentials = array(
    'bearer_token' => xxxxxx,
    'consumer_key' => xxxxxx,
    'consumer_secret' => xxxxxx,
    'token_identifier' => xxxxxx,
    'token_secret' => xxxxxx,
);

//instantiate the object
$twitter = new BirdElephant($credentials);

//get a user's followers using the handy helper methods
$followers = $twitter->user('coderjerk')->followers();

//pass your query params to the methods directly
$following = $twitter->user('coderjerk')->following([
    'max_results' => 20,
    'user.fields' => 'profile_image_url'
]);

// You can also use the sub classes / methods directly if you like:
$user = new UserLookup($credentials);
$user = $user->getSingleUserByID('2244994945', null);`
I changed what should be change with my own bearer_token, secret keys ....
But when I try the script I have this error message 
`Parse error: syntax error, unexpected '/', expecting ';' or ',` 
it's in the first ligne meaning that it at the level of the `use `keyword.

**Expected behavior**
That the script show me the followrs in json

**Environment (please complete the following information):**
 - windows 10
 - 7.4.2
 - last version got it with composer install

**Additional context**
I am noobs, I am prety sure I did somthing wrong and I am sorry to bother you but really need some help

**Comment by danieldevine** [comment]
Hi @dzGeekyGirl,

Hard to know without seeing your code exactly, but try the below and let me know the result you get.

```php

use Coderjerk/BirdElephant/BirdElephant;

$credentials = array(
'bearer_token' => xxxxxx,
'consumer_key' => xxxxxx,
'consumer_secret' => xxxxxx,
'token_identifier' => xxxxxx,
'token_secret' => xxxxxx,
);

$twitter = new BirdElephant($credentials);

try {
    $followers = $twitter->user('coderjerk')->followers();
} catch (Exception $e) {
    var_dump($e->getResponse()->getBody()->getContents());
}

if ($followers) :
    foreach ($followers->data as $follower) :
        echo "<h3>" . $follower->name . "</h3>";
    endforeach;
endif;

```

**Reply by dzGeekyGirl** [reply]
Hi @danieldevine ;
Thank you for your answer.
I tried your code and I have the same error
`Parse error: syntax error, unexpected '/', expecting ';' or ',' in C:\laragon\www\script\twitterApi\twitterApi.php on line 2`
it look like the issue is that it doens't reconize the use
I will give more specification here.
I used composer from WLS2 (ubuntu) to install the tool.
I have this structure in my foloder
twitterApi
  -vendor
    --coderjerk
    --composer
    --guzzlehttp
    ......etc 
  -composer.json
  -composer.lock
  -twitterApi.php
 The code that you give me I put it in twitterApi.php
 I am using laragon with Apach 2, PHP 7.4.2 in windows 10
 thanks again

**Comment by danieldevine** [comment]
A few things to try:

1. Check your php version on **both** the command line and the web server (they can be different) by using `php -v` on the command line and by adding  ```echo phpversion();``` to the first line of your file and seeing what it outputs.
2. Check that you're autoloading from composer properly -  I'm assuming that you have ```require __DIR__ . '/vendor/autoload.php';``` somewhere in your code? In the command line make sure you run `composer dump-autoload` after making changes.

**Reply by dzGeekyGirl** [reply]
hi @danieldevine, 
thank again for your answer
I did what you told me but still have the same issu
the php version in my terminal (using WLS2 with ubuntu on windows 10) is 
`PHP 7.4.3 (cli) (built: Oct 25 2021 18:20:54) ( NTS )
Copyright (c) The PHP Group
Zend Engine v3.4.0, Copyright (c) Zend Technologies
    with Zend OPcache v7.4.3, Copyright (c), by Zend Technologies`
For the excussion of the script I am using Laragon with php 7.4.20-win32-vc15-x64 with apache2.4.35-win64-vc15 standard
When I try to excute the script directly on the terminal (WLS2 ubuntu) 
` php twitterApi.php`
I have the same error on the terminal witch is 
`PHP Parse error:  syntax error, unexpected '/', expecting ';' or ',' in /mnt/c/laragon/www/script/twitterApi/twitterApi.php on line 3`

**Comment by dzGeekyGirl** [comment]
I think I figure it out.
I added 
`require __DIR__ . '/vendor/autoload.php';`
Like @danieldevine suggest.
And changed this
`use Coderjerk/BirdElephant/BirdElephant;`
to this
`use Coderjerk\BirdElephant\BirdElephant;`
mean it was \ in place / 
is it something related to OS ? 
thank you @danieldevine  for you help and answers

**Reply by danieldevine** [reply]
The forward slashes are a typo in the docs - so totally my fault, amazed I didn't spot that :) I'll update the docs now, thanks for bringing it to my attention.
